badgerdb/badgerdb.go

```diff
@@ -105,8 +138,15 @@ type Options struct {
 // DefaultOptions is an Options object with default values.
 // Dir: "BadgerDB", Codec: encoding.JSON
 var DefaultOptions = Options{
-	Dir:   "BadgerDB",
-	Codec: encoding.JSON,
+	Dir:                           "BadgerDB",
+	InMemory:                      false,
+	SyncWrites:                    false,
+	ReadOnly:                      false,
+	Truncate:                      false,
+	MaxCacheSize:                  1 << 30, // 1 GB, 1 024 576 or 2  bytes
```

Comment:
I've seen `x<<y` in the stdlib as well I think, also for "MB" / "GB", but when being precise, "GB" is Gigabyte, which in fact is 1*1000*1000*1000, while what's calculated with 1 << 30 is "GiB" or Gibibyte, which is 1*1024*1024*1024. Source: https://en.wikipedia.org/wiki/Template:Quantities_of_bytes
=> What's actually used by BadgerDB as default?

Also I think the "to the power of" number got lost in the comment: `or 2  bytes`, right?